web-local/tests/connectors/save-anyway.spec.ts

```diff
@@ -37,14 +34,14 @@ test.describe("Save Anyway feature", () => {
     // Click "Save Anyway" button
     await page.getByRole("button", { name: "Save Anyway" }).click();
 
-    // Wait for the editor to appear (more robust than URL-only wait on CI), then assert URL
+    // Wait for navigation to connector file, then for the editor to appear
+    await expect(page).toHaveURL(/.*\/files\/connectors\/.*\.yaml/, {
+      timeout: 60000,
+    });
     const codeEditor = page
       .getByLabel("codemirror editor")
       .getByRole("textbox");
-    await expect(codeEditor).toBeVisible({ timeout: 45000 });
-    await expect(page).toHaveURL(/.*\/files\/connectors\/.*\.yaml/, {
-      timeout: 45000,
-    });
+    await expect(codeEditor).toBeVisible({ timeout: 60000 });
```

Comment:
We need to lower these timeouts from 60s. In the success case, it shouldn't take 60s for these elements to appear. In the failure case, this'll really lengthen the test time. How about 3-5 seconds?

Reply:
https://github.com/rilldata/rill/pull/8389/commits/059e2748aeda227507ce9239dfe8dad0f1cc1e05